### PR TITLE
feat(playground): embbed codemirror-lang-rome-ast

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**
## Summary

Fixes: https://github.com/rome/tools/issues/2984

1. From https://github.com/rome/tools/pull/2974#pullrequestreview-1056350592

## Test Plan
1. Testing locally

### CODE REVIEW

**Comment by IWANABETHATGUY** [comment]
@MichaReiser , I think this is what you want.

**Comment by IWANABETHATGUY** [comment]
I build successfully locally,  I have no idea what's going on with CI, I give up now.
Feel free to close this pull request.

File: website/playground/package.json [hunk 1]
```diff
@@ -3,22 +3,24 @@
...
 	},
 	"dependencies": {
 		"@codemirror/lang-javascript": "^6.0.1",
-		"codemirror-lang-rome-ast": "0.0.4",
-		"@uiw/react-codemirror": "^4.11.4",
-		"@codemirror/view": "6.1.0",
 		"@codemirror/state": "6.1.0",
+		"@codemirror/view": "6.1.0",
+		"@uiw/react-codemirror": "^4.11.4",
+		"codemirror-lang-rome-ast": "file:./lang-rome-ast",
```
Comment (ematipico) [inline website/playground/package.json:23]: pnpm has support for workspaces, which we should use for this  particular case: https://pnpm.io/pnpm-workspace_yaml

File: website/playground/lang-rome-ast/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  "devDependencies": {
+    "@codemirror/buildhelper": "^0.1.16",
+    "@codemirror/language": "^6.0.0",
+    "@codemirror/state": "^6.0.0",
+    "@codemirror/theme-one-dark": "^6.0.0",
+    "@codemirror/view": "6.0.0",
+    "@lezer/generator": "^1.0.0",
+    "codemirror": "^6.0.0",
+    "rome": "0.7.0-next.f6510d6"
```
Comment (ematipico) [inline website/playground/lang-rome-ast/package.json:42]:
```suggestion
    "rome": "next"
```

File: website/playground/lang-rome-ast/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/IWANABETHATGUY/lang-rome-ast"
```
Comment (ematipico) [inline website/playground/lang-rome-ast/package.json:46]: I presume this is a leftover of your repo?

File: website/playground/lang-rome-ast/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
Comment (ematipico) [inline website/playground/lang-rome-ast/.gitignore:2]: So we know they are folders

```suggestion
node_modules/
dist/
```

**Review by ematipico — CHANGES_REQUESTED** [review]
Could you please check the CI? It's failing

File: website/playground/package.json (hunk 1 above)
Comment (IWANABETHATGUY) [reply website/playground/package.json:23]: I will check this out when I have time, converting it to draft now.

**Comment by IWANABETHATGUY** [comment]
This pull request is too messy, I would open it again once I found a solution.

**Pull request closed without merging**